libexec/pyenv-rehash

```diff
@@ -45,8 +45,8 @@ for (( i=1; i<="${PYENV_REHASH_TIMEOUT:-60}"; i++ )); do
     acquired=1
     break
   else
-    # POSIX sleep(1) doesn't provides time precision of subsecond
-    sleep 1
+    # POSIX sleep(1) doesn't provide subsecond precision, but many others do
+    sleep 0.1 2>/dev/null || sleep 1
```

Comment:
This loop expects the sleep to take 1 second. As-written, it will give up in 1/10th of the expected time if `sleep` supports subsecond precision.

Reply:
Valid, point, fixed in 173fa22e0746c4db3de69c5201b21f416987a052.